src/Frozennode/Administrator/AdministratorServiceProvider.php

```diff
@@ -98,7 +103,7 @@ public function register()
 		$this->app['admin_datatable'] = $this->app->share(function($app)
 		{
 			$dataTable = new DataTable($app->make('itemconfig'), $app->make('admin_column_factory'), $app->make('admin_field_factory'));
-			$dataTable->setRowsPerPage($app->make('session.store'), Config::get('administrator::administrator.global_rows_per_page'));
+			$dataTable->setRowsPerPage($app->make('session.store'), Config::get('administrator.global_rows_per_page'));
 
 			return $dataTable;
 		});
```

Comment:
I guess the Html service provider should be registered here so the user don't need to add it manually in the `config/app.php` file.